operator/internal/manifests/ingester.go

```diff
@@ -185,16 +195,19 @@ func NewIngesterStatefulSet(opts Options) *appsv1.StatefulSet {
 
 // NewIngesterGRPCService creates a k8s service for the ingester GRPC endpoint
 func NewIngesterGRPCService(opts Options) *corev1.Service {
+	s := serviceNameIngesterGRPC(opts.Name)
 	l := ComponentLabels(LabelIngesterComponent, opts.Name)
+	a := serviceAnnotations(s, opts.Flags.EnableCertificateSigningService)
 
 	return &corev1.Service{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "Service",
 			APIVersion: corev1.SchemeGroupVersion.String(),
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   serviceNameIngesterGRPC(opts.Name),
-			Labels: l,
+			Name:        serviceNameIngesterGRPC(opts.Name),
+			Labels:      l,
+			Annotations: a,
```

Comment:
Should we inline `l` and `a` and just keep `s` (maybe renamed to `serviceName`) because it is needed twice?